rc/lsp.kak

```diff
@@ -936,9 +936,9 @@ define-command -hidden lsp-show-hover -params 3 -docstring %{
     lsp_diagnostics=$3
     content=$(eval "${kak_opt_lsp_show_hover_format}")
     # remove leading whitespace characters
-    content="${content#"${content%%[![:space:]]*}"}"
+    content="${content##[[:space:]]}"
```

Comment:
This now only trims one space, see `a='   1 2 3'; echo x"${a##[[:space:]]}"x`
The original construct is really tricky, I'm not sure if there's a better way?

Reply:
Ah, I must have misread the explanations on <https://wiki.bash-hackers.org/syntax/pe#substring_removal>.